odoo/addons/test_lint/tests/test_override_signatures.py

```diff
@@ -15,25 +14,53 @@
 
 
 failure_message = """\
-Invalid override of {model}.{method} in {child_module}, {message}.
+Invalid override in {model} of {method}, {message}.
 
 Original definition in {parent_module}:
     def {method}{original_signature}
 
-Incompatible definition in {child_module}:
+Incompatible override definition in {child_module}:
     def {method}{override_signature}"""
 
 
 MODULES_TO_IGNORE = {
     'pos_blackbox_be',  # TODO cannot update to sanitize without certification
 }
-methods_to_sanitize = {
-    method_name
-    for method_name in dir(BaseModel)
-    if not method_name.startswith('_')
-} - {
+METHODS_TO_IGNORE = {
     # Not yet sanitized...
-    'write', 'default_get'
+    'write', 'default_get',
+    # base
+    'action_timer_stop',
+    '_get_eval_context',
+    # mail
+    'message_mail_with_source',
+    'message_new',
+    'message_post_with_source',
+    'message_update',
```

Comment:
Hello, thank you for adding automated tools to catch these common mistakes! 👍
I'm not sure to understand why we ignore most of these. Is it just to avoid conflicts and merge the PR as soon as possible?
Do we plan to make a second pass to fix them and remove them from here?
`message_update` in particular seems very trivial to fix right now, by just renaming the 2 wrong parameters, rather than ignoring it?

Reply:
It is mainly to avoid conflicts and then let different teams check these. They should be cleaned up in the future, but at least merging this PR will prevent future introductions of such overrides.
For `message_update`, since there was quite a few other functions related to mail, I did not fix it.